extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -536,19 +548,22 @@ export class MigrationCutoverDialog {
 
 			this._model.migrationStatus.properties.migrationStatusDetails?.activeBackupSets?.forEach((activeBackupSet) => {
 
-				tableData.push(
-					{
-						fileName: activeBackupSet.listOfBackupFiles[0].fileName,
-						type: activeBackupSet.backupType,
-						status: activeBackupSet.listOfBackupFiles[0].status,
-						dataUploaded: `${convertByteSizeToReadableUnit(activeBackupSet.listOfBackupFiles[0].dataWritten)}/ ${convertByteSizeToReadableUnit(activeBackupSet.listOfBackupFiles[0].totalSize)}`,
-						copyThroughput: (activeBackupSet.listOfBackupFiles[0].copyThroughput / 1024).toFixed(2),
-						backupStartTime: activeBackupSet.backupStartDate,
-						firstLSN: activeBackupSet.firstLSN,
-						lastLSN: activeBackupSet.lastLSN
+				if (this._shouldDisplayBackupFileTable()) {
+					tableData.push(
+						{
+							fileName: activeBackupSet.listOfBackupFiles[0].fileName,
+							type: activeBackupSet.backupType,
+							status: activeBackupSet.listOfBackupFiles[0].status,
+							dataUploaded: `${convertByteSizeToReadableUnit(activeBackupSet.listOfBackupFiles[0].dataWritten)}/ ${convertByteSizeToReadableUnit(activeBackupSet.listOfBackupFiles[0].totalSize)}`,
+							copyThroughput: (activeBackupSet.listOfBackupFiles[0].copyThroughput / 1024).toFixed(2),
+							backupStartTime: activeBackupSet.backupStartDate,
```

Comment:
is the default for copyThroughput 0 or can it be undefined and cause .toFixed to throw an error

Reply:
I added a check to see if activeBackupSet.listOfBackupFiles[0].copyThroughput is null